### PR TITLE
Remove unused code and fix ATAN2 tests

### PR DESCRIPTION
**Pull request by Pankraty**
I accidentally found that some of the classes and methods are not used; also, we had two tests without `[Test]` attribute and bodies not fully matching their names.

### CODE REVIEW

**Comment by igitur** [comment]
Do you perhaps know at what stage these classes become unused? Or were they always unused?

**Comment by Pankraty** [comment]
`XLColumnParameters` and `XLRowParameters` were used for creating XLColumn and XLRow instance before range repository was introduced. `XLStylizedContainer` aimed to be some kind of generic entity having style but apparently was not of any use.

File: ClosedXML/Excel/XLWorkbook.cs [hunk 1]
```diff
@@ -288,45 +288,6 @@ private void InitializeTheme()
             };
         }
 
-        internal XLColor GetXLColor(XLThemeColor themeColor)
```
Comment (igitur) [inline ClosedXML/Excel/XLWorkbook.cs]: Instead of removing this, isn't it useful and should be made public?

Comment (igitur) [reply ClosedXML/Excel/XLWorkbook.cs]: Although I would maybe refactor it to support for future additions:
```c#
public XLColor GetXLColor(XLThemeColor themeColor)
{
    var propertyName = Enum.GetName(typeof(XLThemeColor), themeColor);
    var xlColor = typeof(XLTheme).GetProperty(propertyName)?.GetValue(this.Theme) as XLColor;
    return xlColor ?? throw new ArgumentException("Invalid theme color");
}
```

Comment (igitur) [reply ClosedXML/Excel/XLWorkbook.cs]: Nevermind, I see there is a `XLTheme.ResolveThemeColor` that already does this.

**Pull request merged**